wordpress-mobile/WordPress-Android
# Fix pages in offline mode

Fixes #8521.

![image](https://user-images.githubusercontent.com/1522856/50967885-be057c00-14d9-11e9-9152-59002be3a0a7.png)

### To test:

1. Turn airplane mode ON
2. Go to the Pages screen
3. Now you'll see the overflow menu icon showing for the items in the list
4. Try to move to draft, or move to trash
5. Notice a snackbar message pops up, informing that the device is offline
6. Try to refresh the pages
7. Again, notice a snackbar message pops up, informing that the device is offline
